app/views/sectors/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<h1>Sectors dashboard</h1>
+
+Sectors:
+<%= @sectors_names.map {|s| link_to s.last, sector_path(s.first)}.join(', ').html_safe %>
+<br>
+<br>
+Companies:
+<%= @companies_names.map {|c| link_to c.last, company_path(c.first)}.join(', ').html_safe %>
+<br>
+<br>
+
+<h3>Management Quality: All sectors</h3>
+<p>Distribution of companies in All sectors according to the management of their greenhouse gas emissions and of risks and opportunities to the low-carbon transition.</p>
+
+<div class="chart">
+<%= pie_chart(
+      companies_levels_sectors_path,
```

Comment:
instead of data, we provide URL so it gets fetched after the main page is rendered.